ui-tests/src/test/java/org/openmrs/reference/DeletePersonTest.java

```diff
@@ -31,9 +31,9 @@ public void setup() {
         personUuid = TestData.createPerson(personInfo);
     }
 
-    @Ignore
+	@Test
     @Category(BuildTests.class)
-    public void deletePersonTest() {
+	public void deletePersonTest() throws Exception {
```

Comment:
any reason why this is now necessary yet it wasn't before

Reply:
You definitely want to resolve the test than make it expect an exception to pass